src/styles/variables.scss

```diff
@@ -12,6 +12,12 @@ $light-dark: rgba($dark, 0.88);
 $layout-spacing-small: 20px;
 $layout-spacing-medium: 40px;
 $layout-spacing-large: 95px;
+$border-radius-medium: 10px;
+$border-radius-small: 4px;
+$border-radius-map: (
+	small: $border-radius-small,
+	medium: $border-radius-medium,
+);
```

Comment:
Why do you think you need those separate (15/16) and in a map (17)?

Reply:
I needed the `$border-radius-small `variable in another map, but I remove the border-radius-map and only leave the variables. Since it doesn't make sense to have a map in this case.